app/cypress/e2e/cif/project-revision/undo-changes.cy.js

```diff
@@ -92,7 +92,7 @@ describe("when undoing, the project revision page", () => {
     }).click();
     cy.addOrEditEmissionIntensityReport(
       "2022-01-01",
-      "2022-02-02",
+      "2022-04-02",
```

Comment:
The datepicker was ending up with March instead of February both locally and in CI, so I tried a different date and that seemed to make it happier